src/dbnode/storage/types.go

```diff
@@ -1009,7 +1009,7 @@ type databaseMediator interface {
 
 // OnColdFlush can perform work each time a series is flushed.
 type OnColdFlush interface {
-	ColdFlushNamespace(ns Namespace) (OnColdFlushNamespace, error)
+	ColdFlushNamespace(ns Namespace, reuseResources bool) (OnColdFlushNamespace, error)
```

Comment:
Can we make this an options interface? So it's easier to extend/add additional options in the future.

Reply:
Sure. Do you mean `ColdFlushNamespace(ns Namespace, opts SomeOptions)`?

Reply:
Either works, no preference for me :).